### ISSUE TITLE
Color

### ISSUE THREAD

**Issue by sicay1** [vanessa-luna/banner-comments-plus#7]
It would be nice if can display banner comment in color, similar to this [Better Comments](https://github.com/aaron-bond/better-comments).

**Comment by vanessa-luna** [comment]
I agree that would be nifty. And should I get time this winter and the energy I will implement. Thanks for the idea.
